examples/kernel_approximation/plot_scalable_poly_kernels.py

```diff
@@ -148,15 +148,15 @@
 ksvm_score = 100 * ksvm.score(X_test, y_test)
 
 results["KSVM"] = {"time": ksvm_time, "score": ksvm_score}
-print(f"Kernel-SVM score on raw featrues: {ksvm_score:.2f}%")
+print(f"Kernel-SVM score on raw features: {ksvm_score:.2f}%")
 
 # %%
 # Finally, plot the results of the different methods against their training
 # times. As we can see, the kernelized SVM achieves a higher accuracy,
 # but its training time is much larger and, most importantly, will grow
 # much faster if the number of training samples increases.
 
-N_COMPONENTS = [250, 500, 1000, 2000]
+N_COMPONENTS = [250, 450, 850, 1750]
```

Comment:
Actually you should not have to redefine this variable. It could be reused from the previous cells.